migrations/20231011151001-change-org-self-auth-to-boolean.js

```diff
@@ -0,0 +1,13 @@
+const TABLE = 'organization';
+
+exports.up = db => Promise.all([
```

Comment:
The `Promise.all` method runs the promises in the array concurrently and not in succession. Since some of the SQL calls depend on other calls, you'll want to either use the `async/await` syntax to guarantee the calls are called in order.

Reply:
Ie. https://github.com/cloud-gov/pages-core/blob/4268-change-self-auth-to-boolean/migrations/20210224210834-init-organizations.js#L78-L91

Reply:
**Great** catch, @apburnes. Order is certainly important here. Thank you!